src/strings/PowerFxResources.en-US.resx

```diff
@@ -1022,6 +1022,44 @@
   <data name="AboutISOWeekNum_date" xml:space="preserve">
     <value>A date value for which the ISO week number will be calculated.</value>
   </data>
+  <data name="AboutEDate" xml:space="preserve">
+    <value>Returns the date adjusted by a number of months.  The date portion of the result remains the same but is capped by the number of days in the new month.</value>
+    <comment>Description of 'EDate' function.</comment>
+  </data>
+  <data name="EDateArg1" xml:space="preserve">
+    <value>date</value>
+    <comment>function_parameter - First parameter for the EDate function - a date value to be adjusted by the specified number of months.</comment>
+  </data>
+  <data name="EDateArg2" xml:space="preserve">
+    <value>months</value>
+    <comment>function_parameter - Second (optional) parameter for the EDate function - Number of months by which to change the date. A positive value yields a future date, a negative value a past date, and a zero will not change the month. Translate this string. Maintain as a single word (do not add spaces).</comment>
+  </data>
+  <data name="AboutEDate_date" xml:space="preserve">
+    <value>A date value to be adjusted by the specified number of months.</value>
+  </data>
+  <data name="AboutEDate_months" xml:space="preserve">
+    <value>Number of months by which to change the date. A positive value yields a future date, a negative value a past date, and a zero will not change the month.</value>
+    <comment>{Locked=Months}</comment>
+  </data>
+  <data name="AboutEOMonth" xml:space="preserve">
+    <value>Returns the date adjusted by a number of months.  The date portion of the result is the last day of the new month.</value>
```

Comment:
>Returns the date adjusted by a number of months [](http://example.com/codeflow?start=11&length=47)

Copy/paste issue - incorrect description

Reply:
Not sure I see this one, but I may have already fixed it with other changes.  Is it still here?